tests/otel_postgres_metrics_e2e/test_postgres_metrics.py

```diff
@@ -0,0 +1,111 @@
+import os
+import re
+from utils import context, scenarios
+
+
+@scenarios.otel_postgres_metrics_e2e
+class Test_PostgreSQLMetricsCollection:
+
+    def test_postgresql_metrics_comprehensive_validation(self):
+        collector_log_path = f"{context.scenario.host_log_folder}/docker/collector/stderr.log"
+        assert os.path.exists(collector_log_path), f"Collector log file not found: {collector_log_path}"
+
+        with open(collector_log_path, 'r', encoding='utf-8') as f:
+            log_content = f.read()
+
+        POSTGRESQL_METRICS = {
```

Comment:
TODO add the Optional Metrics: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/postgresqlreceiver/documentation.md